### PR TITLE
Update commands.md

### PR DESCRIPTION
**Pull request by dongemus**
Typo fixes
Corrected Big Karp weight
Added !unraid

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -71,8 +71,8 @@ available filters:
 |atk        |`!track eevee atk15`             | Tracks eevee with a (minimum) 15 point attack value|
 |def        |`!track eevee def14`             | Tracks eevee with a (minimum) 14 point defense value|
 |sta        |`!track eevee sta10`             | Tracks eevee with a (minimum) 10 point stamina value|
-|weight     |`!track magikarp weight131300`   | Tracks "big" magikarp (131300 grams and higher|
-|maxweight  |`!track rattata maxweight24100`  | Tracks "tiny" rarrara (24100 grams and lower)|
+|weight     |`!track magikarp weight13130`   | Tracks "big" magikarp (13130 grams and higher|
+|maxweight  |`!track rattata maxweight24100`  | Tracks "tiny" rattata (24100 grams and lower)|
```
Comment (fosJoddie) [inline docs/commands.md]: rattata should be `!track rattata maxweight2410`

File: docs/commands.md [hunk 2]
```diff
@@ -97,6 +97,10 @@ Available filters:
 |valor|`!raid moltres valor`| Tracks moltres raids where controlling team is valor|
 |harmony|`!raid venusaur harmony`| Tracks venusaur raids in uncontested gyms|
 
+###### !unraid
+
+`!unraid snorlax` Removes tracking of Snorlax raids.
```
Comment (fosJoddie) [inline docs/commands.md]: add also the way to remove tracking per raid tier: "`!unraid level5` Removes tracking of tier 5 raids"

File: docs/commands.md [hunk 3]
```diff
@@ -199,6 +203,9 @@ Available filters:
 |valor|`!channel raid moltres valor`| Tracks moltres raids where controlling team is valor|
 |harmony|`!channel raid venusaur harmony`| Tracks venusaur raids in uncontested gyms|
 
+###### !unraid
+
+`!channel unraid snorlax` Removes tracking of Snorlax raids.
```
Comment (fosJoddie) [inline docs/commands.md]: add also the way to remove tracking per raid tier: "`!channel unraid level5` Removes tracking of tier 5 raids"

File: docs/commands.md [hunk 4]
```diff
@@ -175,8 +179,8 @@ available filters:
 |atk        |`!channel track eevee atk15`             | Tracks eevee with a (minimum) 15 point attack value|
 |def        |`!channel track eevee def14`             | Tracks eevee with a (minimum) 14 point defense value|
 |sta        |`!channel track eevee sta10`             | Tracks eevee with a (minimum) 10 point stamina value|
-|weight     |`!channel track magikarp weight131300`   | Tracks "big" magikarp (131300 grams and higher|
-|maxweight  |`!channel track rattata maxweight24100`  | Tracks "tiny" rarrara (24100 grams and lower)|
+|weight     |`!channel track magikarp weight13130`   | Tracks "big" magikarp (13130 grams and higher|
+|maxweight  |`!channel track rattata maxweight24100`  | Tracks "tiny" rattata (24100 grams and lower)|
```
Comment (fosJoddie) [inline docs/commands.md]: adjust rattata maxweight to `2410`

**Comment by KartulUdus** [comment]
Thanks guys 😊 
Made @fosJoddie corrections and rebased on develop with fb4e28e2493d7a77b8d5169b4aae0e203d808249

Please open PR against Develop in the future; however, you might notice a role has been added to you in the discord @dongemus

**Pull request closed without merging**